Hack for downversion fontconfig that has no support for variation fonts

--- webkitgtk-2.42.0/Source/cmake/OptionsGTK.cmake~	2023-09-15 08:27:19.481914800 +0000
+++ webkitgtk-2.42.0/Source/cmake/OptionsGTK.cmake	2023-09-15 18:11:18.779116311 +0000
@@ -14,7 +14,7 @@
 set(USER_AGENT_BRANDING "" CACHE STRING "Branding to add to user agent string")
 
 find_package(Cairo 1.16.0 REQUIRED)
-find_package(Fontconfig 2.13.0 REQUIRED)
+find_package(Fontconfig 2.12.1 REQUIRED)
 find_package(Freetype 2.9.0 REQUIRED)
 find_package(LibGcrypt 1.6.0 REQUIRED)
 find_package(HarfBuzz 1.4.2 REQUIRED COMPONENTS ICU)
--- webkitgtk-2.42.0/Source/cmake/OptionsGTK.cmake~	2023-09-15 18:11:18.779116311 +0000
+++ webkitgtk-2.42.0/Source/cmake/OptionsGTK.cmake	2023-09-15 18:55:51.900081373 +0000
@@ -149,7 +149,7 @@
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SERVICE_WORKER PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SHAREABLE_RESOURCE PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SPEECH_SYNTHESIS PRIVATE ${ENABLE_EXPERIMENTAL_FEATURES})
-WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_VARIATION_FONTS PRIVATE ON)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_VARIATION_FONTS PRIVATE OFF)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_API_STATISTICS PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_CODECS PRIVATE ${ENABLE_EXPERIMENTAL_FEATURES})
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_RTC PRIVATE ${ENABLE_EXPERIMENTAL_FEATURES})
